crates/hyle-loadtest/src/lib.rs

```diff
@@ -470,7 +470,7 @@ pub async fn long_running_test(node_url: String, _indexer_url: String) -> Result
             // Create a new user
             if now % 5 == 0 || users.len() < 2 {
                 let ident = Identity(format!("{}.{}", now, random_hydentity_contract.0));
-                users.push(now);
+                users.push(now.try_into().unwrap());
```

Comment:
We could maybe turn users into something like
```suggestion
                let mut users: Vec<u128> = vec![];
```
to prevent the try&unwrap. 
Didn't check if it totally fits everywhere though